doc/operation-and-maintenance/Mongoose-metrics.md

```diff
@@ -102,42 +106,45 @@ All metrics are divided into following groups:
+To see the `hook <-> XMPP metric` translation please refer to [mongoose_metrics_hooks](https://github.com/esl/MongooseIM/blob/exometer/apps/ejabberd/src/mongoose_metrics_hooks.erl#L71) file.
  
 ## Global metrics description
 
 | Metric name | Type | Description |
 | ----------- | ---- | ----------- |
-| nodeSessionCount | function | Number of sessions on given Erlang node |
+| nodeSessionCount | function | Number of sessions on a given Erlang node |
 | totalSessionCount | function | Total number of sessions in the cluster |
 | uniqueSessionCount | function | Number of unique sessions in the cluster |
 
 ### Definitions
-* `session`: it is user and resource
-* `unique session`: only user, without resource. E.g if a user is connected to the server from 3 different resources, it will be counted 3 times in the session counter but only once in the unique sessions counter.
+* `session`: a user and a resource
+* `unique session`: only a user, without the resource. 
+ E.g if a user is connected to the server from 3 different resources, they will be counted 3 times in the session counter but only once in the unique sessions counter.
 
-## Backends metrics description
+## Backend metrics description
 
-Existence of this kind of metrics is driven by the status of corresponding module.
-For example if mod_roster is enabled, relevant `backends` metrics for this module are created and updated.
-This kind of metrics are histograms and contain information about execution time of certain functions from backend module.
+The existence of this kind of metrics is driven by the status of the corresponding module.
+For example if mod_roster is enabled, the relevant `backend` metrics for this module are created and updated.
+This kind of metrics are histograms and contain information about the execution time of certain functions from the backend module.
 
-The only `backends` metrics which are always present are related to authentication module as this one is always enabled.
+The only `backend` metrics which are always present are related to the authentication module as this one is always enabled.
 
 ## Data metrics description
 
 | Metric name | Type | Description |
 | ----------- | ---- | ----------- |
-| [xmpp,received,xml_stanza_size] | histogram | Received (by the server) stanza size stats after possible decryption and decompression |
-| [xmpp,sent,xml_stanza_size] | histogra | Sent (to the client) stanza size stats before possible encryption and compression |
-| [xmpp,received,compressed_size] | histogram | Received compressed stanza size stats |
-| [xmpp,sent,compressed_size] | histogram | Sent compressed stanza size stats |
-| [xmpp,received,encrypted_size] | histogram | Received encrypted stanza size stats |
-| [xmpp,sent,encrypted_size] | histogram | Sent encrypted stanza size stats|
+| [xmpp,received,xml_stanza_size] | histogram | Size stats of a stanza received by the server after a possible decryption and decompression |
```

Comment:
Maybe this is enough:
`Size stats of a received plain stanza` and similar for the other side.